### DISCUSSION TITLE
add magento's option to prestashop - combinations are joined to long description

### DISCUSSION THREAD

**Discussion by Maofree** [PrestaShop/PrestaShop#40184] [Ideas]
Hi

go to this site made with magento
https://www.giordanoshop.com/piatto-doccia-in-pietra-70x90-cm-bonussi-ardesia-stone-side-bianco.html

if you change the combination, some data inside the long description will change

![picturemessage_dk23gyvl wc4](https://user-images.githubusercontent.com/653230/59196334-a4528880-8b8e-11e9-9cd9-3cf039bf00cd.png)
![picturemessage_gxvj4jas 4zx](https://user-images.githubusercontent.com/653230/59196336-a4eb1f00-8b8e-11e9-9623-a084fd07f8f2.png)
![3](https://user-images.githubusercontent.com/653230/59196343-a9173c80-8b8e-11e9-979c-3218ae08fb47.png)

the same thing could be done for the specifics data
prestashop should be able to handle more complex products, where it is necessary to have a more detailed and specific configuration, in this way the information seen by customers would be the right one for the combination chosen

thanks

**Comment by khouloudbelguith** [comment]
Hi @maofree,

Thanks for your report.
@marionf @colinegin, what do you think? can we add a new feature so the description will be customized for each combination?

Thanks!

**Comment by ghost** [comment]
I think it's not a core Magento feature. This can be done in PrestaShop quite easily with custom template and some JS. 

BTW. Mentioned site is sooo overloaded by the scripts... 
BTW 2. What kind of improvement it might be? I do not see the practical benefit of using this solution

**Comment by Maofree** [comment]
Hi @PrestaShark 
I've checked better, yes you are right, they change only the combination value in the description no other changes is done inside the description.

however there are many sites that could be necessary to have different text for each combination, this feature could be appreciated by different companies, because this way they could have specific data for each combination, and the visitor would avoid misinterpreting the descriptions, in practice it would all be clearer

bye

**Comment by matks** [comment]
@maofree If I understand correctly, either they have 1 description for each combination, or they use a description with a placeholder inside like `{ combination.data }` that is being replaced when switching from one combination to another ?

**Comment by Maofree** [comment]
hi
the same change is done on the main title

![1](https://user-images.githubusercontent.com/653230/59341738-accece80-8d08-11e9-9b18-217436e64dbd.png)

**Comment by Maofree** [comment]
now I'm trying to understand if this is a feature of magento or not
I'm looking about other sites
my customer notify me this feature, saying it is a magento's feature but I'm not sure

**Comment by Maofree** [comment]
in this site the title doesn't change so I think that is a customization
@matks that value you could put it at the end of the title using { combination.data }, also because in the middle without using javascript code I don't think it's possible

**Comment by ghost** [comment]
@maofree i still claim my opinion that this is not a core Magento feature - it's pure custom. I was able to make the same in Prestashop 1.7 (few months ago) with custom .js code and a bit work with default variables - when customer change color or size that values/combinations was added to product name. 

I think that its not crucial function to add right now in Prestashop roadmaps. To make it good PS folks would have to rework a lot of code to push that name (with added combination/s to cart and checkout. A lot of mess and additional bugs and glitches.

Even in mentioned site product name not correspond with url (in Prestashop does ;) )
![ui](https://user-images.githubusercontent.com/8243273/59354791-17dace00-8d26-11e9-83be-0ae1efb82cef.jpg)

**Comment by ghost** [comment]
![ui2](https://user-images.githubusercontent.com/8243273/59355422-3b524880-8d27-11e9-95e5-3009a55e8f22.jpg)
